hubblestack/extmods/fileserver/s3fs.py

```diff
@@ -440,6 +444,9 @@ def __get_s3_meta(bucket, key=s3_key_kwargs['key'], keyid=s3_key_kwargs['keyid']
                                         path_style=s3_key_kwargs['path_style'],
                                         https_enable=s3_key_kwargs['https_enable'],
                                         params={'marker': marker})
+            if not tmp:
+                return
```

Comment:
Let's have this explicitly `return None`, to avoid `inconsistent returns` (a return statement returns something, the other returns empty) even though it still technically returns None